File: chapter_3/Account/Account.py
class Account:
    def __init__(self, initial_deposit=0):
        self.balance = initial_deposit

    def withdrawFunds(self, amount: float) -> float:
        if amount > self.balance:
            amount = self.balance

        self.balance -= amount

        return amount

    def depositFunds(self, amount: float):
        self.balance += amount


if __name__ == "__main__":
    a = Account(250.0)
    b = Account(999.0)
    c = Account(8850.0)

    print("I have three accounts.")
    print(f"Account 1 has the prope balance of {a.balance}.")
    print(f"Account 2 has the prope balance of {b.balance}.")
    print(f"Account 3 has the prope balance of {c.balance}.")

File: chapter_3/Account/Account.java
public class Account {
    private double balance;

    public Account () {}

    public Account (double amount) {
        this.balance = amount;
    }

    public double withdrawFunds(double amount) {
        if (amount > this.balance) {
            amount = this.balance;
        }

        this.balance -= amount;
        return amount;
    }

    public void depositFunds(double amount) {
        this.balance += amount;
    }

    public double getBalance() {
        return this.balance;
    }

    public static void main (String [] args)  {
        Account a = new Account(250.0);
        Account b = new Account(999.0);
        Account c = new Account(8850.0);

        System.out.println("I have three accounts.");
        System.out.println("Account 1 has the prope balance of " + a.getBalance() + ".");
        System.out.println("Account 2 has the prope balance of " + b.getBalance() + ".");
        System.out.println("Account 3 has the prope balance of " + c.getBalance() + ".");
    }
}
